Extensions/editable_reblogs.js

```diff
@@ -56,6 +56,9 @@ XKit.extensions.editable_reblogs = new Object({
 		XKit.interface.post_window.set_content_html(all_quotes_text + old_content);
 
 		$(".btn-remove-trail .icon").click();
+		$(".post-form--form > div > div").filter(function(index, element) {
+		    return $(this).data("subview") === "reblogTree";
+		}).css("display", "none");
```

Comment:
I think this can be simplified to `$('.control-reblog-trail.removed') rather than dealing with the filter function.